.github/workflows/build_library.yml

```diff
@@ -0,0 +1,33 @@
+name: Binary Release Build
+
+on:
+  push:
+    branches:
+      - master
+      - release/**
+
+
+jobs:
+  linux:
+    name: Linux
+    runs-on: ubuntu-latest
+  
+    steps:
+      - uses: actions/checkout@v2
+    
+      - name: Set up docker for build (x86_64)
```

Comment:
TODO: We should probably use a cache to avoid rebuilding this image all the time.

Reply:
Will look into how to do this.

Reply:
If we use Kaniko, it does this automatically. Kaniko doesn't have an official GitHub action though so we can simply pull the "latest" builder image which automatically populates common layers. Then we push it back to Docker Hub as "cache".

Reply:
> Then we push it back to Docker Hub as "cache".

Do you mean we should push from this workflow or should we do that via cloudbuild ?